config/model_configs/plane_agnesi_mountain_test_stretched.yml

```diff
@@ -10,8 +10,8 @@ x_elem: 80
 dz_bottom: 200.0
 use_reference_state: false
 config: "plane"
-kappa_4_vorticity: 1.0e6
-kappa_4_tracer: 1.0e6
+kappa_4_vorticity: 0.001
```

Comment:
Ditto. Why is it so much smaller than the other coefficients?

Reply:
@akshaysridhar ?

Reply:
The original values for the plane configurations are conservative estimates, based on the suggested coefficients for box configuration test cases in `https://gmd.copernicus.org/articles/9/2007/2016/gmd-9-2007-2016.pdf`. (We haven't tuned these plane/box test coefficients further since these cases were introduced).

Reply:
Eq. (72) in this paper actually gives coefficients around 1. But it seems we should be able to run this simulation without hyperdiffusion. I will turn off hyperdiffusion for this case.

Reply:
Oh, wait, I think you are using Eq. (77), which gives 0.001.